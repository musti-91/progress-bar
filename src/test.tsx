import * as React from "react";

import ScrollProgress from ".";

describe("ScrollProgress", () => {
  /**
   * snapshot: [default]
   */
  it("should match snapshot default", () => {
    expect(<ScrollProgress />).toMatchSnapshot();
  });
  /**
   * snapshot [props]
   */
  it("should match snapshot with Props", () => {
    expect.assertions(1);

    const props = {
      rtl: "left",
      width: "30%",
      height: "100px",
      color: "green"
    };

    expect(<ScrollProgress {...props} />).toMatchSnapshot();
  });
});
