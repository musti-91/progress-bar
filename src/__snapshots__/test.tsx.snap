// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollProgress should match snapshot default 1`] = `<ScrollProgress />`;

exports[`ScrollProgress should match snapshot with Props 1`] = `
<ScrollProgress
  color="green"
  height="100px"
  rtl="left"
  width="30%"
/>
`;
